docs/installation.md

```diff
@@ -5,7 +5,7 @@
 You can install CRI Resource Manager from `deb` or `rpm` packages
 for supported distros.
 
-  - [download](https://github.com/intel/cri-resource-manager/releases) packages
+  - [download](https://github.com/intel/cri-resource-manager/releases/tag/v0.4.1) packages
```

Comment:
So, before we make a point release, are we supposed to change the docs to point to a tag that does not exist yet ? Then tag the release ? And if something goes wrong, update the docs again and repeat ?